src/Core/Content/Category/Event/SalesChannelEntryPointsEvent.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Category\Event;
+
+use Shopware\Core\System\SalesChannel\SalesChannelEntity;
+
+class SalesChannelEntryPointsEvent
+{
+    /**
+     * @var array
+     */
+    protected $ids;
+
+    /**
+     * @var SalesChannelEntity
+     */
+    protected $salesChannelEntity;
+
+    public function __construct(array $ids, SalesChannelEntity $salesChannelEntity)
```

Comment:
is it possible to add the whole SalesChannelContext here?

Reply:
Not really, as the event is also used in places where the saleschannelcontext is not readily available.